README.md

```diff
@@ -82,27 +82,13 @@ Please go to [./docs/classes/ethereumhdkey.md](./docs/classes/ethereumhdkey.md)
 
 ## Provider Engine
 
-The Wallet can be easily plugged into [provider-engine](https://github.com/metamask/provider-engine) to provide signing:
+Provider Engine is
+[not very actively maintained](https://github.com/MetaMask/web3-provider-engine#web3-providerengine)
+and support has been removed along `v1.0.0` release, see
+issue [#115](https://github.com/ethereumjs/ethereumjs-wallet/issues/115) for context.
 
-Node.js / ES5:
-
-```js
-const { WalletSubprovider } = require('ethereumjs-wallet')
-
-<engine>.addProvider(new WalletSubprovider(<wallet instance>))
-```
-
-ESM / TypeScript:
-
-```js
-import { WalletSubprovider } from 'ethereumjs-wallet'
-
-<engine>.addProvider(new WalletSubprovider(<wallet instance>))
-```
-
-Note it only supports the basic wallet. With a HD Wallet, call `getWallet()` first.
-
-Please go to [./docs/classes/walletsubprovider.md](./docs/README.md) for more info.
+You can use the the old `src/provider-engine.ts` code (see associated PR) as some boilerplate
+for your own integration if needed.
 
 ### Remarks about `toV3`
```

Comment:
This header level is a bit confusing here, should it be 2 levels deep instead? I don't think it's meant to be nested under `## Provider Engine`.

Otherwise, PR LGTM!

Reply:
(The README could generally tolerate some additional love at some PIT.)